Automattic/wp-calypso
# Plugins: Remove Business plan plugins nudge A/B test

This pull request removes code from the A/B test introduced in 12184-gh-calypso-pre-os that displays a nudge on the `Plugins` page:

![screenshot](https://cloud.githubusercontent.com/assets/594356/12787245/e42f2882-ca94-11e5-8388-73e59848eae8.png)

#### Testing instructions
 
1. Run `git checkout remove/business-plugins-nudge-test` and start your server
2. Log into a test account with a site that hasn't been upgraded to a plan yet
3. Open the [`Plugins` page](https://wpcalypso.wordpress.com/plugins) on `wpcalypso`
4. Execute `localStorage.setItem('ABTests','{"businessPluginsNudge_20151119":"nudge"}')` in your browser's console
5. Reload the page and checks that the nudge is displayed
6. Now open the [`Plugins` page](http://calypso.localhost:3000/plugins) on your machine
7. Execute `localStorage.setItem('ABTests','{"businessPluginsNudge_20151119":"nudge"}')` in your browser's console
8. Reload the page and checks that the nudge is not displayed and that no error is thrown
 
#### Reviews
